packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.schema.cue

```diff
@@ -0,0 +1,790 @@
+package dashboard
+
+DashboardV2: {
+  kind: "Dashboard"
+  spec: DashboardSpec
+}
+
+DashboardSpec: {
+  // Unique numeric identifier for the dashboard.
+  // `id` is internal to a specific Grafana instance. `uid` should be used to identify a dashboard across Grafana instances.
+  id?: int64
+
+  // Title of dashboard.
+  title: string
+
+  // Description of dashboard.
+  description?: string
+
+  // Configuration of dashboard cursor sync behavior.
+  // "Off" for no shared crosshair or tooltip (default).
+  // "Crosshair" for shared crosshair.
+  // "Tooltip" for shared crosshair AND shared tooltip.
+  cursorSync: DashboardCursorSync
+
+  // When set to true, the dashboard will redraw panels at an interval matching the pixel width.
+  // This will keep data "moving left" regardless of the query refresh rate. This setting helps
+  // avoid dashboards presenting stale live data.
+  liveNow?: bool
+
+  // When set to true, the dashboard will load all panels in the dashboard when it's loaded.
+  preload: bool
+
+  // Whether a dashboard is editable or not.
+  editable?: bool | *true
+
+  // Links with references to other dashboards or external websites.
+  links: [...DashboardLink]
+
+  // Tags associated with dashboard.
+  tags?: [...string]
+
+  timeSettings: TimeSettingsSpec
+
+  // Configured template variables.
+  variables: [...QueryVariableKind | TextVariableKind | ConstantVariableKind | DatasourceVariableKind | IntervalVariableKind | CustomVariableKind | GroupByVariableKind | AdhocVariableKind]
+
+  elements: [ElementReference.name]: PanelKind // |* more element types in the future
+
+  annotations: [...AnnotationQueryKind]
+
+  layout: GridLayoutKind
+
+  // version: will rely on k8s resource versioning, via metadata.resorceVersion
+
+  // revision?: int // for plugins only
+  // gnetId?: string // ??? Wat is this used for?
+}
+
+
+AnnotationPanelFilter: {
+  // Should the specified panels be included or excluded
+  exclude?: bool | *false
+
+  // Panel IDs that should be included or excluded
+  ids: [...uint8]
+}
+
+// "Off" for no shared crosshair or tooltip (default).
+// "Crosshair" for shared crosshair.
+// "Tooltip" for shared crosshair AND shared tooltip.
+DashboardCursorSync: "Off" | "Crosshair" | "Tooltip"
```

Comment:
```suggestion
DashboardCursorSync: "off" | "crosshair" | "tooltip"
```

we use lowercase in other places :)

Reply:
Cog automatically decides this. It always uses tittle case. It is probably a standard way to generate it. @K-Phoen, do you have any recommendations here?